sisyphus/global_settings.py

```diff
@@ -130,8 +130,10 @@ def file_caching(path):
 JOB_CLEANER_INTERVAL = 60
 #: How many threads should be cleaning in parallel
 JOB_CLEANER_WORKER = 5
-#: If the job internal work directory should be keeped re deleted during clean up
+#: If the job internal work directory should be kept or deleted during clean up
 JOB_CLEANUP_KEEP_WORK = False
+#: If the job internal input directory with symlinks to input jobs should be kept or deleted during clean up
+JOB_CLEANUP_KEEP_INPUT = False
```

Comment:
I would even vote for having this be enabled by default (if this breaks nothing)

Reply:
What happens if `input` is kept and is also in the `finished.tar.gz` and then you unzip the `finished.tar.gz`? Will that lead to an error because the dir `input` and its file already exists?

Reply:
But it should not be in the zipped file if its kept right?

Reply:
Currently it is never in the zipped file, even if it is not kept. I thought this was supposed to be changed. I'm not sure how it is then. So you say it should not be in the zipped file if it is kept?

Reply:
Yes in this PR you are right. But I would change this behavior to only be included in the zip if the flag is not set. I guess my other comment kind of included that to me, but I did not make this clear.